app/.eslintrc

```diff
@@ -1,6 +1,53 @@
 {
   "rules": {
     "no-alert": "off",
-    "no-console": "off"
+    "no-console": "off",
+    "import/order": [
+      "error",
+      {
+        "groups": [
+          ["external", "builtin"],
+          "internal",
+          ["parent", "sibling", "index"]
+        ],
+        "pathGroups": [
+          {
+            "pattern": "react",
+            "group": "external",
+            "position": "before"
+          },
+          {
+            "pattern": "assets/**",
+            "group": "internal"
+          },
+          {
+            "pattern": "components/**",
+            "group": "internal"
+          },
+          {
+            "pattern": "generator/**",
+            "group": "internal"
+          },
+          {
+            "pattern": "lib/**",
+            "group": "internal"
+          },
+          {
+            "pattern": "pages/**",
+            "group": "internal"
+          },
+          {
+            "pattern": "types/**",
+            "group": "internal"
+          }
+        ],
+        "pathGroupsExcludedImportTypes": ["react"],
+        "newlines-between": "always",
+        "alphabetize": {
+          "order": "asc",
+          "caseInsensitive": true
+        }
```

Comment:
I personally don't have a strong opinion on this, but... the fact that there are so many changes makes me think this is a bit too opinionated.

Some statistics:
- With the current config, there are `1780 errors`
- Without `alphabetize`, it'd be `831 errors`
- With `"newlines-between": "always-and-inside-groups"` (a bit more permissive), `637 errors`
- Without this block, `145 errors`

Reply:
I'll remove the `alphabetize` block to reduce the number of errors. It was just a personal preference since the code feels more "clean" with it.